charts/newrelic-infrastructure/values-windows-linux.yaml

```diff
@@ -0,0 +1,14 @@
+global:
+  licenseKey: <> 
+  cluster: <> 
+
+kubelet:
+  enableLinux: true
```

Comment:
i couldn't find anywhere that uses this value in this PR. is this needed?

Reply:
Removed this label.